pkg/syncer/syncer.go

```diff
@@ -59,8 +59,8 @@ type RedisNotificationSyncer struct {
 	sdkKey   string
 }
 
-// NewRedisNotificationSyncer returns an instance of RedisNotificationSyncer
-func NewRedisNotificationSyncer(logger *zerolog.Logger, conf config.SyncConfig, sdkKey string) (*RedisNotificationSyncer, error) {
+// NewRedisSyncer returns an instance of RedisNotificationSyncer
+func NewRedisSyncer(logger *zerolog.Logger, conf config.SyncConfig, sdkKey string) (*RedisSyncer, error) {
```

Comment:
Looks good. I had something like this in mind. We can share the same syncer.
I approve it as is. We can discuss more after datafile-syncer is added. Then we may have a better idea :)

```
NewNotificationHandler(syncer: RedisSyncer, config, sdkKey) {
    ...
    send(event) {
        syncer.send(event.message)
    }
}
```